src/Loads.js

```diff
@@ -46,8 +51,8 @@ export default class Loads extends Component<Props, State> {
   _paused: boolean;
 
   state = {
-    error: this.props.error,
-    response: this.props.response
+    error: idx(this.props, _ => _.cache.error),
```

Comment:
idx > lodash, idx provides type inference through this syntax, whereas lodash's `get` does not (since it's string based)